Topics/Tech_Stacks/GDB_Debug.md

```diff
@@ -12,8 +12,6 @@ GDB (GNU Debugger) is a powerful open-source debugger commonly used in the field
 
 It is a command-line tool that allows developers to inspect and debug programs written in various programming languages, including C, C++, and even some other languages.
 
-Source: [GDB: The GNU Project Debugger](https://www.sourceware.org/gdb/)
```

Comment:
Actually keep the link but not as a separate line

Reply:
Alright, Thanks for your suggestion.